playbook-dev.yaml

```diff
@@ -7,13 +7,13 @@ content:
   sources:
   - url: .
     start_paths: content/*
-  - url: .
-    start_paths: ['localLinks/AxonFramework/docs/*', '!localLinks/AxonFramework/docs/_playbook']
-  - url: ./localLinks/AxonSynapse
-    start_paths: ['/docs/*', '!/docs/_playbook']
-    branches: [development, 'synapse-*']
-  - url: .
-    start_paths: ['localLinks/giftcard-demo/docs/*', '!localLinks/giftcard-demo/docs/_playbook']
+ # - url: .
+ #   start_paths: ['localLinks/AxonFramework/docs/*', '!localLinks/AxonFramework/docs/_playbook']
+  #- url: ./localLinks/AxonSynapse
+  #  start_paths: ['/docs/*', '!/docs/_playbook']
+  #  branches: [development, 'synapse-*']
+  #- url: .
+  #  start_paths: ['localLinks/giftcard-demo/docs/*', '!localLinks/giftcard-demo/docs/_playbook']
```

Comment:
I guess you know that, but by commenting those out, you build only the content in this repo, and thus, that is the only content the prose linter is checking. 

You really need those to check the errors in the other repos.